app/services/hawkular_proxy_service.rb

```diff
@@ -92,16 +92,31 @@ def tenants(limit)
     tenants = @cli.hawkular_client.http_get('/tenants')
 
     if @params['include'].blank?
-      tenants.map! { |x| x["id"] }
+      tenants.map! { |x| {:label => humanize(x["id"]), :value => x["id"]} }
     else
-      tenants.map! { |x| x["id"] if x["id"].include?(@params['include']) }
+      tenants.map! { |x| {:label => humanize(x["id"]), :value => x["id"]} if x["id"].include?(@params['include']) }
     end
 
     tenants.compact[0..limit]
   end
 
   private
 
+  def humanize(id)
+    special_cases = {
+      "_system" => "System",
+      "_ops"    => "Operations"
+    }
+
+    if special_cases.key?(id)
```

Comment:
@yaacov why all this complex logic. If it's in the `special_cases` (rename and make it a constant) then use that, otherwise just use the plain hawkular id.